packages/costa/src/runtime.ts

```diff
@@ -363,7 +363,12 @@ export class CostaRuntime {
         '--default-timeout=1000',
         `--cache-dir=${this.options.installDir}/.pip`
       ]);
-      return spawnAsync(`${envDir}/bin/pip3`, args, {}, stdio);
+      return spawnAsync(`${envDir}/bin/pip3`, args, {
```

Comment:
Using [`--install-option`](https://pip.pypa.io/en/stable/reference/pip_install/#install-install-option) might be a standard way.